docs/source/terms_and_model.rst

```diff
@@ -1147,3 +684,965 @@ The GA4GH computed identifier for these sets is
+regulatory, transcribed, and/or other functional Locations.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - gene_id
+     - :ref:`CURIE`
+     - 0..1
+     - Authoritative Gene ID (see guidance)
+   * - type
+     - string
+     - 1..1
+     - MUST be set to **`Gene`**
+
+**Implementation guidance**
+
+* Gene symbols (e.g., "BRCA1") are unreliable keys.  Implementations
+  MUST NOT use a gene symbol to define a Gene.
+* A gene is specific to a species.  Gene orthologs have distinct
+  records in the recommended databases.  For example, the BRCA1 gene
+  in humans and the Brca1 gene in mouse are orthologs and have
+  distinct records in the previously recommended gene databases.
+* Implementations MUST use authoritative gene namespaces available from
+  identifiers.org whenever possible.  Examples include:
+
+    * `hgnc <https://registry.identifiers.org/registry/hgnc>`__
+    * `ncbigene <https://registry.identifiers.org/registry/ncbigene>`__
+    * `ensembl <https://registry.identifiers.org/registry/ensembl>`__
+    * `vgnc <https://registry.identifiers.org/registry/vgnc>`__
+    * `mgi <https://registry.identifiers.org/registry/mgi>`__
+* The `hgnc` namespace is RECOMMENDED for human
+  variation in order to improve interoperability.
+* Gene MAY be converted to one or more :ref:`Locations <Location>`
+  using external data. The source of such data and mechanism for
+  implementation is not defined by this specification.
+
+**Example**
+
+The following examples all refer to the human BRCA1 gene:
+
+.. parsed-literal::
+
+   {
+     'gene': 'ncbigene:672',
+     'type': 'Gene'
+   }
+
+   {
+     'gene': 'hgnc:1100',
+     'type': 'Gene'
+   }
+
+   {
+     'gene': 'ensembl:ENSG00000012048',
+     'type': 'Gene'
+   }
+
+
+**Sources**
+
+* `SequenceOntology: gene (SO:0000704)
+  <http://www.sequenceontology.org/browser/current_release/term/SO:0000704>`__
+  — A region (or regions) that includes all of the sequence elements
+  necessary to encode a functional transcript. A gene may include
+  regulatory regions, transcribed regions and/or other functional
+  sequence regions.
+
+.. _Quantity:
+
+Quantity
+@@@@@@@@
+
+A value indicating a multitude or magnitude measure.
+
+.. _CopyCount:
+
+CopyCount
+#########
+
+**Computational Definition**
+
+An integer count of copies. Counts are bounded ranges
+denoted by minimum and maximum possible values.
+Absolute copy number counts may not be smaller than zero.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - type
+     - string
+     - 1..1
+     - MUST be "CopyCount"
+   * - absolute_measure
+     - bool
+     - 1..1
+     - specifies if the count is an absolute (TRUE)
```

Comment:
Suggest using True, False instead of TRUE, FALSE as used in other bool desc e.g. derived sequence